medusa/providers/torrent/html/bithdtv.py

```diff
@@ -190,12 +186,7 @@ def login(self):
             self.session.cookies.clear()
             return False
 
-        if '<h2>Login failed!</h2>' in response.text:
-            log.warning('Invalid username or password. Check your settings')
-            self.session.cookies.clear()
-            return False
-
-        return True
+        return self.validate_cookie_login(response, '<h2>Login failed!</h2>')
```

Comment:
No don't think so. I did that at first to make it more explicit. but I can use just something like Login Failed I asume.